content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -0,0 +1,121 @@
+remove the grep `/var/run/docker.sock` to review other mounts.
+
+```bash
+kubectl get pods --all-namespaces \
+-o=jsonpath='{range .items[*]}{"\n"}{.metadata.namespace}{":\t"}{.metadata.name}{":\t"}{range .spec.volumes[*]}{.hostPath.path}{", "}{end}{end}' \
+| sort \
+| grep '/var/run/docker.sock'
+```
+
+{{< note >}}
+There are alternative ways for a pod to access Docker on the host. For instance, the parent
+directory `/var/run` may be mounted instead of the full path (like in [this
+example](https://gist.github.com/itaysk/7bc3e56d69c4d72a549286d98fd557dd)).
+The script above only detects the most common uses.
+{{< /note >}}
+
+### Detecting Docker dependency from node agents
+
+In case your cluster nodes are customized and install additional security and
+telemetry agents on the node, make sure to check with the vendor of the agent whether it has dependency on Docker.
+
+## Telemetry and security agent vendors
+
+This section is intended to collect information about various telemetry and
+security agents that may have dependency on containers runtime.
+
+### SignalFx (Splunk)
+
+The [SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html)
+Smart Agent uses several different monitors for Kuberentes including
+`kubernetes-cluster`, `kubelet-stats`/`kubelet-metrics`, and
+`docker-container-stats`. Monitor `kubelet-stats` is already deprecated for
+Kubernetes <1.18 in favor of `kubelet-metrics`. `Docker-container-stats` is the
+one affected by dockershim removal. It should NOT be used with other runtimes.
+
+How to migrate from Dockershim-dependant agent:
+
+1. Remove `docker-container-stats` from the list of
+   [configured monitors](https://docs.signalfx.com/en/latest/integrations/agent/monitors/_monitor-config.html).
+   Note, keeping this monitor enabled with non-dockershim runtime will result in
+   incorrect metrics being reported when docker is installed on node and no
+   metrics when docker is not installed.
+2. [Enable and configure](https://docs.signalfx.com/en/latest/integrations/agent/monitors/_monitor-config.html)
+   [`kubelet-metrics`](https://docs.signalfx.com/en/latest/integrations/agent/monitors/kubelet-metrics.html)
+   monitor.
+
+Note, the set of collected metrics will change. Please review your
+[alerting rules](https://docs.signalfx.com/en/latest/detect-alert/) and
+[dashboards](https://docs.signalfx.com/en/latest/dashboards/index.html).
+
+The pod accessing Docker may be named like:
+- `signalfx-agent`
+
+### Other agents
+
+We keep the work in progress version of migration instructions in
+[Google doc](https://docs.google.com/document/d/1ZFi4uKit63ga5sxEiZblfb-c23lFhvy6RXVPikS8wf0/edit#).
+
+{{< note >}}
+
+Vendors, please help to fill out this section. See
+[CONTRIBUTING](https://github.com/kubernetes/website/blob/master/CONTRIBUTING.md)
+guide for instructions. Or create an issue if you need help formatting the
+document.
+
+{{< /note >}}
```

Comment:
(this section is not complete; also, I don't think we'd accept any of those PRs)

Reply:
Do you have any thoughts on what would be the appropriate number of vendor docs would be for the initial version of the document? Is there any prior art in this repo we can learn from?

Reply:
> the appropriate number of vendor docs 

Given https://github.com/kubernetes/enhancements/tree/master/keps/sig-docs/1326-third-party-content-in-docs#readme, zero. We can link to vendor documentation for this sort of thing but we don't host it.

Reply:
I removed vendor docs and kept the link to the google doc